Description: Use Debian normaliz package instead of normaliz shipped with M2.
 Also backport change from upstream to work with normaliz 3.0+.
Author: Doug Torrance <dtorrance@piedmont.edu>
Origin: https://github.com/Normaliz/Normaliz/commit/22fab00
Last-Update: 2016-04-11

--- a/M2/Macaulay2/packages/Normaliz.m2
+++ b/M2/Macaulay2/packages/Normaliz.m2
@@ -156,19 +156,7 @@
 -- get the normaliz executable with full path
 getNmzExec=()->
 (
-    if(nmzVersion!="")
-    then(
-        if(nmzVersion!="norm64" and nmzVersion !="normbig" and nmzVersion!="normaliz") then <<error("nmzVersion must be one of the following: normaliz normbig norm64")
-        else(
-        nmzExec:=nmzVersion;
-        );
-    )
-    else
-    (
-        nmzExec="normaliz";
-    );
---    return nmzExec;
-    return prefixDirectory | currentLayout#"programs" | nmzExec;
+     return "normaliz";
 );
 
 
--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -1013,11 +1013,6 @@
      BUILD_csdp=yes
 fi
 
-dnl we can't use the test below for the presence of normaliz until the package Normaliz knows
-dnl how to find normaliz on the path, and until our *.dmg and
-dnl *.deb making procedures know how to include it or set the dependencies appropriately
-BUILD_normaliz=yes
-BUILD_ALWAYS="$BUILD_ALWAYS normaliz"
 dnl AC_MSG_CHECKING(whether the package normaliz is installed)
 dnl if test "`type -t normaliz`" = file
 dnl then AC_MSG_RESULT(yes)
